lib/make-spawn-args.js

```diff
@@ -30,7 +30,8 @@ const makeSpawnArgs = options => {
     npm_config_node_gyp,
   })
 
-  const fileName = escape.filename(`${event}-${Date.now()}`)
+  const rand = require('crypto').randomBytes(4).toString('hex')
```

Comment:
we should put a `const { randomBytes } = require('crypto')` at the top of this file instead of the late require here. i think that would also make the line below this one short enough that we can embed the call to randomBytes in the template string

Reply:
Resolved https://github.com/npm/run-script/pull/95/files#diff-db13c26a02b6b712f679d9378d8970ae22553f8a84ebd716d4c498121f350b52